### PR TITLE
Change way we make branch name safe, add original branch name

### PR DESCRIPTION
**Pull request by ricca509**
☝️

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -42,12 +42,13 @@ async function run(): Promise<void> {
       branchName = process.env.GITHUB_REF?.split('/')
         .slice(2)
         .join('/')
-        .replace(/\//g, '-')
+        .replace(/[^0-9a-z]/g, '-')
```
Comment (ccouzens) [inline src/main.ts:45]: What if the branch contains upper case characters?

Comment (ricca509) [reply src/main.ts:45]: should be ok?

Comment (ricca509) [reply src/main.ts:45]: As in it will not prevent it from working

Comment (ricca509) [reply src/main.ts:45]: In fact, Alex gave us an example already 🤣

**Review by ccouzens — APPROVED** [review]

**Pull request merged**